cpu/cortexm_common/vectors_cortexm.c

```diff
@@ -203,29 +217,17 @@ __attribute__((used)) void hard_fault_handler(uint32_t* sp, uint32_t corrupted,
     if( corrupted ) {
         puts("Stack pointer corrupted, reset to top of stack");
     } else {
```

Comment:
I know it's unrelated but can you adapt the three lines above to the coding conventions while you're at it?